context/app/static/js/components/Routes.jsx

```diff
@@ -65,6 +66,14 @@ function Routes(props) {
       </Container>
     );
   }
+
+  if ('markdown' in flaskData) {
+    return (
+      <Container maxWidth="lg">
+        <Markdown markdown={flaskData.markdown} />
```

Comment:
Destructure at the top similar to other flaskData entries?

Reply:
Sure!